website/docs/d/event_orchestrations.html.markdown

```diff
@@ -37,9 +37,9 @@ The following arguments are supported:
 * `name_filter` - The list of the Event Orchestrations which name match `name_filter` argument.
```

Comment:
I don't see this in the published docs https://registry.terraform.io/providers/PagerDuty/pagerduty/latest/docs/data-sources/event_orchestration, nor do I see `name_filter` in  I'm not sure why it says `name_filter` in the [code definition](https://github.com/PagerDuty/terraform-provider-pagerduty/blob/master/pagerduty/data_source_pagerduty_event_orchestration.go) for the event orchestration data source so I'm not sure why this markdown file says `name_filter`. Maybe I'm editing the wrong markdown file?

Reply:
Yeah, this is a bit confusing because we have two data sources that look very similar:
- `pagerduty_event_orchestrations`: lets you pull a list of orchestrations matching the given name filter. This is where it's [documented](https://registry.terraform.io/providers/PagerDuty/pagerduty/latest/docs/data-sources/event_orchestrations) and [defined in the code](https://github.com/PagerDuty/terraform-provider-pagerduty/blob/master/pagerduty/data_source_pagerduty_event_orchestrations.go)
- `pagerduty_event_orchestration`: returns a single orchestration matching the given name. It's documented and defined in the links that you posted.